### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by Inego**

### CODE REVIEW

File: docs/exception-handling.md [hunk 1]
```diff
@@ -428,19 +428,19 @@ fun main() = runBlocking {
         supervisorScope {
             val child = launch {
                 try {
-                    println("Child is sleeping")
+                    println("The child is sleeping")
```
Comment (qwwdfsad) [inline docs/exception-handling.md:431]: Please also run `./gradlew knit` to update actual `.kt` files

File: docs/coroutine-context-and-dispatchers.md [hunk 2]
```diff
@@ -625,7 +625,7 @@ fun main() = runBlocking<Unit> {
 In this example we launch a new coroutine in a background thread pool using [Dispatchers.Default], so
 it works on a different thread from the thread pool, but it still has the value of the thread local variable
 that we specified using `threadLocal.asContextElement(value = "launch")`,
-no matter on what thread the coroutine is executed.
+no matter what thread the coroutine is executed on.
```
Comment (LouisCAD) [inline docs/coroutine-context-and-dispatchers.md:628]:
```suggestion
no matter which thread the coroutine is executed on.
```

Comment (Inego) [reply docs/coroutine-context-and-dispatchers.md]: Yes, thanks.

File: docs/exception-handling.md (hunk 1 above)
Comment (Inego) [reply docs/exception-handling.md:431]: It doesn't work, it says `JDK_16 environment variable is not defined.`

Comment (Inego) [reply docs/exception-handling.md:431]: It's really, really sad that I would have to install JDK 1.6 just to fix some typos :(

Comment (LouisCAD) [reply docs/exception-handling.md:431]: If the "Allow modifications from maintainers" checkbox is checked, maybe @qwwdfsad can do it?

**Comment by elizarov** [comment]
Thanks. I've run knit and merged manually via f4fb204b220b224d5ffa9b7c00ff077b681213e9 Btw, for the purposes of knit you can simply set JDK_16 to JDK_HOME. I'll see if I can patch the code to make it happen automatically.

**Comment by Inego** [comment]
Thank you, Roman!

**Pull request closed without merging**